### DISCUSSION TITLE
Why was Default vs New York removed from website?

### DISCUSSION THREAD

**Discussion by lambegraham** [shadcn-ui/ui#5835] [Q&A]
Previously on the website theme section you could choose between Default vs New York and see the difference.  
![image](https://github.com/user-attachments/assets/533ab539-dde0-4601-9c2c-f1714fcd1da6)

Now there is no longer an option to see between Default vs New York. Was this removed by mistake? 
<img width="1624" alt="Screenshot 2024-11-14 at 22 31 44" src="https://github.com/user-attachments/assets/da9f7922-d9de-41a5-8c85-b52fffa24c5a">

**Comment by KingPsychopath** [comment]
Bump!
